src/core/issues-notes.js

```diff
@@ -146,42 +137,46 @@ function handleIssues(ins, ghIssues, conf) {
             })
             .map(createLabel)
             .reduce((frag, labelElem) => {
-              frag.appendChild(labelElem);
+              frag.append(labelElem);
               return frag;
             }, document.createDocumentFragment());
-          $tit.append(
-            $(
-              "<span style='text-transform: none'>: " + report.title + "</span>"
-            ).append(labelsGroup)
-          );
-          $inno.removeAttr("title");
+          const node = hyperHTML`<span style='text-transform: none'>: ${
+            report.title
+          }</span>`;
+          node.append(labelsGroup);
+
+          tit.append(node);
+          inno.removeAttribute("title");
         } else if (report.title) {
-          $tit.append(
-            $(
-              "<span style='text-transform: none'>: " + report.title + "</span>"
-            )
+          tit.append(
+            hyperHTML`<span style='text-transform: none'>: ${
+              report.title
+            }</span>`
           );
-          $inno.removeAttr("title");
+          inno.removeAttribute("title");
         }
-        $tit.addClass("marker");
-        $div.append($tit);
-        $inno.replaceWith($div);
-        let body = $inno.removeClass(report.type).removeAttr("data-number");
-        if (ghIssue && !body.text().trim()) {
-          body = ghIssue.body_html;
+        tit.classList.add("marker");
+        div.append(tit);
+        let body = inno;
+        inno.replaceWith(div);
+        body.classList.remove(report.type);
+        body.removeAttribute("data-number");
+        if (ghIssue && !body.innerHTML.trim()) {
+          body = hyperHTML`${ghIssue.body_html}`;
         }
-        $div.append(body);
-        const level = $tit.parents("section").length + 2;
-        $tit.attr("aria-level", level);
+        div.append(body);
+        const level = getCount(tit);
+        tit.setAttribute("aria-level", level);
       }
       pub(report.type, report);
     });
-  if ($(".issue").length) {
-    if ($("#issue-summary"))
-      $("#issue-summary").append($issueSummary.contents());
-  } else if ($("#issue-summary").length) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+  if (document.querySelectorAll(".issue").length) {
+    if (issueSummaryElement)
+      issueSummaryElement.innerHTML = issueSummary.innerHTML;
+  } else if (issueSummaryElement) {
     pub("warn", "Using issue summary (#issue-summary) but no issues found.");
-    $("#issue-summary").remove();
+    issueSummaryElement.parentNode.removeChild(issueSummaryElement);
```

Comment:
You can instead do `issueSummaryElement.remove()` even without jQuery 😀

Reply:
lol, it gets confusing sometimes. Thanks!